openmls/src/group/mls_group/creation.rs

```diff
@@ -77,12 +80,15 @@ impl MlsGroup {
     ///
     /// Note: If there is a group member in the group with the same identity as
     /// us, this will create a remove proposal.
+    #[allow(clippy::too_many_arguments)]
     pub fn join_by_external_commit<Provider: OpenMlsProvider>(
         provider: &Provider,
         signer: &impl Signer,
         ratchet_tree: Option<RatchetTreeIn>,
         verifiable_group_info: VerifiableGroupInfo,
         mls_group_config: &MlsGroupJoinConfig,
+        capabilities: Option<Capabilities>,
+        extensions: Option<Extensions>,
```

Comment:
A slightly nicer solution might be to have a new config for external joins (`MlsGroupJoinExternalConfig`?) and make these two configurable there.

Reply:
I agree, I created https://github.com/openmls/openmls/issues/1607 for that purpose.